README.md

```diff
@@ -11,4 +10,142 @@ UAVCAN is an open lightweight protocol designed for reliable communication in ae
 robust vehicle bus networks.
 
 * [**UAVCAN website**](http://uavcan.org)
-* [**UAVCAN mailing list**](https://groups.google.com/forum/#!forum/uavcan)
+* [**UAVCAN forum**](https://forum.uavcan.org)
+
+## Identifier ranges
+
+### Subjects
+
+For message subjects, the following range mapping is adopted (limits inclusive).
+Unused ranges are reserved for future expansion of adjacent ranges.
+
+From    | To        | Purpose
+--------|-----------|------------------------------------------------
+0       | 0         | Not a valid ID
```

Comment:
What is the intended use case for an invalid ID?

Reply:
Representing an unconfigured ID.

E.g., a node may have a configuration parameter selecting which Subject ID to use for a particular function. Setting it to zero means that the function is disabled.

Reply:
It really sounds analogous to the mistake of `NULL`.

I think it's **much** more valuable to be certain that every `u16` is a valid subject ID. 

Having a `bool enabled` would be better code and less prone to error than having to check `subject_id != 0` everywhere.

This would be a DSDL convetion, meaning it would also be a breach of abstraction to use it in the core uavcan implementation.

Reply:
Makes sense. Reverted this change back.